express-notes-api/delete.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function deleteEntry(app, data, path, fs, writeForDelete) {
+  app.delete('/api/notes/:id', (req, res) => {
```

Comment:
It would be best to set up all your endpoints in one file and then leave this file to just do the delete functionality. In general, you want functions to just one thing. This one's doing two things: setting up the endpoint and performing the delete action.